test/public/runs/detail.test.js

```diff
@@ -305,15 +299,13 @@ module.exports = () => {
         await goToPage(page, 'run-detail', { queryParameters: { runNumber: 105 } });
 
         await pressElement(page, '#edit-run');
-        await waitForTimeout(100);
         expect(await page.$('#runQualitySelect')).to.be.null;
```

Comment:
Here same, you need to wait for something

Reply:
Done